readme.markdown

```diff
@@ -3,6 +3,7 @@
 tap-producing test harness for node and browsers
 
 [![build status](https://secure.travis-ci.org/substack/tape.svg?branch=master)](http://travis-ci.org/substack/tape)
+[![codecov](https://codecov.io/gh/substack/tape/branch/master/graph/badge.svg)](https://codecov.io/gh/substack/tape)
```

Comment:
it'd be great if this worked, to avoid having to hardcode the default branch name (it works everywhere on github.com). Any chance yall could support that?
```suggestion
[![codecov](https://codecov.io/gh/substack/tape/branch/HEAD/graph/badge.svg)](https://codecov.io/gh/substack/tape)
```